src/App.js

```diff
@@ -248,6 +264,7 @@ export default class App extends React.Component<{}, AppState> {
                         <Col md={8} lg={9}>
                             <ProgramBlockEditor
                                 activeProgramStepNum={this.state.activeProgramStepNum}
+                                actionButtonDisabled={this.state.modalIsShowing}
```

Comment:
Rather than a new property called `actionButtonDisabled` let's make the property the same name as the App state value: `modalIsShowing`. This will communicate that it is the ProgramBlockEditor's responsibility to render itself appropriately when a modal is being displayed.